CanalTP/Chaos
# Add management of empty array error in json validator

# Description

This PR change the error message to override the default validator message "[] is too short"

## Issue

Issue link: bot-749

## How to test

Here are the following steps to test this pull request:

- POST / PUT disruption and POST / PUT impact with "impacts":[]
- POST / PUT disruption and POST / PUT impact with "objects":[]
- POST / PUT disruption and POST / PUT impact with "application_periods":[]
... (you can use also all array that have this validation : minItems = 1 -> see chaos\formats.py)
